sources/packages/forms/src/form-definitions/institutionwelcomepage.json

```diff
@@ -0,0 +1,3175 @@
+                    },
+                    "allowCalculateOverride": false,
+                    "encrypted": false,
+                    "showCharCount": false,
+                    "showWordCount": false,
+                    "allowMultipleMasks": false,
+                    "addons": [],
+                    "tree": false,
+                    "lazyLoad": false,
+                    "components": [
+                        {
+                            "label": "HTML",
+                            "labelWidth": "",
+                            "labelMargin": "",
+                            "tag": "div",
+                            "className": "",
+                            "attrs": [
+                                {
+                                    "attr": "",
+                                    "value": ""
+                                }
+                            ],
+                            "content": "Need help?",
+                            "refreshOnChange": false,
+                            "customClass": "category-header-large primary-color mt-5 mb-5",
+                            "hidden": false,
+                            "modalEdit": false,
+                            "key": "html21",
+                            "tags": [],
+                            "properties": {},
+                            "conditional": {
+                                "show": null,
+                                "when": null,
+                                "eq": "",
+                                "json": ""
+                            },
+                            "customConditional": "",
+                            "logic": [],
+                            "attributes": {},
+                            "overlay": {
+                                "style": "",
+                                "page": "",
+                                "left": "",
+                                "top": "",
+                                "width": "",
+                                "height": ""
+                            },
+                            "type": "htmlelement",
+                            "input": false,
+                            "tableView": false,
+                            "placeholder": "",
+                            "prefix": "",
+                            "suffix": "",
+                            "multiple": false,
+                            "defaultValue": null,
+                            "protected": false,
+                            "unique": false,
+                            "persistent": false,
+                            "clearOnHide": true,
+                            "refreshOn": "",
+                            "redrawOn": "",
+                            "dataGridLabel": false,
+                            "labelPosition": "top",
+                            "description": "",
+                            "errorLabel": "",
+                            "toolt
```

Comment:
I believe we are not using this hidden content "applicationStatus" in our form, can we please remove it?